src/content/codelabs/iterables.md

```diff
@@ -1216,7 +1216,7 @@ If you want to learn more,
 here are some suggestions for where to go next:
 
 * Play with [DartPad.]({{site.dartpad}})
-* Try another [codelab](/codelabs).
+* Try another [tutorial](/tutorials).
 * Read the [Iterable API reference][iterable class]
   to learn about methods not covered by this codelab.
```

Comment:
**issue:** 

* Same as [line 2778](https://github.com/dart-lang/site-www/pull/5858/files#diff-75ed5ce4cf7ea66ddd2f5aa89546afe28498b1afb33bcf6a6f56fb7407445cd6R2778) in [js-to-dart.md](src/content/resources/coming-from/js-to-dart.md)
* Condition, then instruction per [GSG](https://developers.google.com/style/sentence-structure).
* Write in active voice per [GSG](https://developers.google.com/style/voice).

```suggestion
  to learn about methods that this tutorial doesn't.
```